specutils/tests/test_spectrum1d.py

```diff
@@ -167,6 +168,8 @@ def test_wcs_transformations():
     assert isinstance(pix_axis, np.ndarray)
     assert isinstance(disp_axis, u.Quantity)
 
+    assert np.allclose(spec.wcs.world_to_pixel([7000*u.AA]), [461.2])
```

Comment:
shouldn't this be `[7000]*u.AA` instead of `[7000*u.AA]` ?  Or if this is intentionally supposed to be the latter, maybe it should instead be something like `[7000*u.AA, 700*u.nm]` and then make sure those give the same pixel coordinate?